Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -507,16 +511,21 @@ static CGFloat effectivePointsPerMeter(CALayer *caLayer)
     else
         REEntitySetName(m_model->rootEntity(), "WebKit:ModelRootEntity");
 
+    [m_rkContainerEntity setTransform:WKEntityTransform({ simd_make_float3(1.0f, 1.0f, 1.0f), simd_quaternion(0, simd_make_float3(1, 0, 0)), simd_make_float3(0.0f, 0.0f, -m_originalBoundingBoxExtents.z / 2.0f) })];
```

Comment:
Could use some help breaking this line down a little, like splitting the S and R and T components into local vars. Otherwise it takes quite a bit of brainpower to understand what this line is doing.